source/includes/steps-install.yaml

```diff
@@ -0,0 +1,67 @@
+title: "Install Xcode from the App Store."
+ref: install-xcode
+content: |
+  
+  Install Xcode -- it takes awhile. In the meantime, you can follow the rest of the steps.
+
+---
+title: Download and install Homebrew
+ref: install-homebrew
+content: |
+
+  You can get homebrew `here <https://brew.sh>`_.
+
+  Then run:
+
+  .. code-block:: sh 
+   
+     /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+  
+  When this completes, run the following 2 commands:
+
+  .. code-block:: sh
+
+     brew update 
+     brew doctor
+
+---
+title: Install Python 2 and Python 3
+ref: install-python
+content: |
+  
+  To run the build tools you need both python and python 2. Install them both by running:
+
+  .. code-block:: sh
+
+     brew install python python2
+
+  You will also need Pip, the Python package manager. Note you may need to run this in a new shell:
```

Comment:
Nit: Avoid future tense: "The build tools also require Pip, ..."

Reply:
done